src/lib/utils/wallets.ts

```diff
@@ -62,10 +62,18 @@ export const getWalletPersonas = (wallet: WalletData) => {
 // Get a list of wallet supported languages with native title
 export const getSupportedLanguages = (
   walletSupportedLanguages: string[],
-  languages: {}[]
+  languages: {}[],
+  locale: string
 ) => {
   const supportedLanguages = [] as string[]
 
+  // current locale should appear first on the list, this manipulates the array to move it to the top if needed
+  const localeIndex = walletSupportedLanguages.findIndex(
```

Comment:
Won't this affect any other implementations we have that use this function?

Reply:
This util is only used to get the supported languages per wallet